src/ui/digital-exchange/components/list/ComponentList.js

```diff
@@ -1,13 +1,17 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Spinner } from 'patternfly-react';
+import { Spinner, Row, Col } from 'patternfly-react';
+import moment from 'moment';
+import { formattedText } from '@entando/utils';
+import { renderInstallActions, renderComponentImageOrPlaceholder } from 'helpers/digital-exchange/components';
+import StarRating from 'ui/digital-exchange/common/StarRating';
 
 class ComponentList extends Component {
```

Comment:
What about splitting this component in two different ones? One for the list view mode and another for the grid view mode